verifiers/verifier.go

```diff
@@ -74,3 +75,12 @@ func VerifyNpmPackage(ctx context.Context,
 	return verifier.VerifyNpmPackage(ctx, attestations, tarballHash,
 		provenanceOpts, builderOpts)
 }
+
+// VerifyVSA verifies the VSA attestations.
+func VerifyVSA(ctx context.Context,
+	attestations []byte,
+	vsaOpts *options.VSAOpts,
+	verificationOpts *options.VerificationOpts,
+) ([]byte, *utils.TrustedAttesterID, error) {
```

Comment:
I can't seem to tell where these returned values are consumed? `[]byte` and the `trustedAttesterID`? Can we update the comment?

Reply:
the `[]byte` is consumed when doing `PrintAttestation`, and for anyone using the `VerifyVSA()` as a library function.

Reply:
+1, comment should document what the returned values are.

Reply:
added